src/DotNet/ModuleDefMD.cs

```diff
@@ -235,6 +235,19 @@ public sealed class ModuleDefMD : ModuleDefMD2, IInstructionOperandResolver {
 			return Load(mod, new ModuleCreationOptions(context), imageLayout);
 		}
 
+		static IntPtr GetModuleHandle(System.Reflection.Module mod) {
+#if NETSTANDARD2_0
+			var methods = typeof(Marshal).GetMethods();
```

Comment:
There should be a GetMethod(string, Type[]) you can call. You should verify that it takes a Module as its single parameter (in case they add overloads in the future)